src/Simplex/Messaging/Crypto/ShortLink.hs

```diff
@@ -18,6 +29,13 @@ import Simplex.Messaging.Agent.Protocol
 import qualified Simplex.Messaging.Crypto as C
 import Simplex.Messaging.Encoding
 import Simplex.Messaging.Protocol (EntityId (..), LinkId, EncDataBytes (..), QueueLinkData)
+import Simplex.Messaging.Util (liftEitherWith)
+
+fixedDataPaddedLength :: Int
+fixedDataPaddedLength = 2008 -- 2048 - 24 (nonce) - 16 (auth tag)
+
+userDataPaddedLength :: Int
+userDataPaddedLength = 13784 -- 13824 - 24 - 16
```

Comment:
where do 2048 and 13824 come from?

Reply:
1-time invitation link with PQ keys is ~1500 bytes, if there will be redundancy here or some other part in fixed data - it reserves ~500 bytes.

contact address is ~130 bytes, but it'll have to be re-worked in the future and will be the same.

Reply:
13824 - just some round number (13.5kb), leaving ~200 bytes for notification credentials.

They are easy to change if some data needs to be added, as the receiving end will accept any size.

It also means that we need to compress profiles.